libs/ui-toolkit/src/components/icon/vega-icons/svg-icons/icon-closed.tsx

```diff
@@ -16,7 +16,7 @@ export const IconClosed = ({
       >
         <path
           d="M0 10C0 4.47715 4.47715 0 10 0C15.5228 0 20 4.47715 20 10C20 15.5228 15.5228 20 10 20C4.47715 20 0 15.5228 0 10Z"
-          fill="#323339"
+          fill="#65676b"
```

Comment:
we shouldn't be using fill properties as these icons should rely on currentColor. See how the other icons are set up.

If this is for the background circle that should be outside of the svg and created/styled with a div/span

Reply:
this is not the current color, it is for the background, and they seemed as different icons altogether in the designs. 
will see what i can do to just reuse ones we have with the background added

Reply:
The icon should not contain the circle background, only the icon itself. None of the icons in the set use a background color right now (I think), we should keep this consistent. If you need a background then it should be style on the wrapping element

Reply:
updated!